ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -46,18 +46,20 @@ function convertToValid(key, model) {
       parseFloat(key.mapSouth || model.get('mapSouth')) + minimumDifference
   }
   if (key.mapNorth !== undefined) {
-    key.mapNorth = Math.max(-90 + minimumDifference, key.mapNorth)
+    key.mapNorth = Math.max(-90, key.mapNorth)
```

Comment:
I'm guessing the minimumDifference logic is no longer necessary given all the changes we've made in this area over the years.  Might be worth removing all of the references.  Although make sure to test all the location input styles and verify that there are no weird issues at the extremes.